core/deployment/src/main/java/io/quarkus/deployment/pkg/PackageConfig.java

```diff
@@ -25,6 +25,12 @@
     @ConfigItem(defaultValue = "false")
     public boolean uberJar;
 
+    /**
+     * If the java runner jar should contain implementation details information in MANIFEST
+     */
+    @ConfigItem(defaultValue = "false")
+    public boolean implementationEntries;
```

Comment:
I wonder if we will end up having more options for the manifest so I think I would make `manifest` a proper config group and then have a `addImplementationEntries` boolean in this config group.

Reply:
And after giving it some more thoughts (and changing my mind), I would say let's make it true by default. People who don't want to expose it will be able to disable it. That's OK IMHO.

Reply:
No problem.